.travis.yml

```diff
@@ -57,6 +57,18 @@ script:
 - ./test.sh
 - if [ "$TRAVIS_OS_NAME" = "linux" ]; then PATH=$PATH:$(pwd) ./test_e2e.sh; fi
 
+jobs:
+  include:
+    - stage: Go Coverage
+      before_script:
+        - go get github.com/mattn/goveralls
+      script:
+        - make test-coverage
+        # todo(camilamacedo86): Add env var with repotoken value in trevis. It should not be a public info. Then, replace as follows
+        # todo(camilamacedo86): ReGen the token in https://coveralls.io/github/kubernetes-sigs/kubebuilder
+        # - $GOPATH/bin/goveralls -service=travis-ci -coverprofile=coverage-all.out -repotoken=$COVERALLS_TOKEN
+        - $GOPATH/bin/goveralls -service=travis-ci -coverprofile=coverage-all.out -repotoken=xaBu9eWjZIkVgBBAe5uLt2Y5IsATjrALo
```

Comment:
To add the env var I will need your help since I have no permission in the Trevis.